app/assets/stylesheets/signup.scss

```diff
@@ -6,11 +6,11 @@
   max-width: 80%;
 }
 
-.form-container {
-  margin-left: 40px;
-  margin-top: 30px;
-  max-width: 80%;
-}
+// .form-container {
+//   margin-left: 40px;
+//   margin-top: 30px;
+//   max-width: 80%;
+// }
```

Comment:
✂️

Reply:
This stylesheet as a whole will be removed